### PR TITLE
Removed Stairs Decals (Depreceated) Portet CM Medical Decals

### PR DESCRIPTION
**Pull request by MaloTV**
## About the PR

New Decals

## Why / Balance
Removed Depreceated Decals due to not being usefull and taking up alot of the decal Pannel
Added Original Cm Medical decals for mapping purposes

## Media
image

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes
New Decals

**Changelog**

:cl:Malo
- add: Added Original CM Medical Decals

### CODE REVIEW

File: Resources/Textures/Decals/medical_decals.rsi/meta.json [hunk 1]
```diff
@@ -0,0 +1,88 @@
+{
+  "version": 1,
+  "license": null,
+  "copyright": null,
```
Comment (QuietlyWhisper) [inline Resources/Textures/Decals/medical_decals.rsi/meta.json:4]: Missing contents here.

**Comment by UbaserB** [comment]
why do i like these decals a lot

**Review by QuietlyWhisper — CHANGES_REQUESTED** [review]
These should be in Textures/.CM14/Decals/Medical
and Prototypes/.CM14/Decals/Medical

All images should be named `name_name.png` for consistency.

File: Resources/Textures/Decals/medical_decals.rsi/meta.json (hunk 1 above)
Comment (QuietlyWhisper) [reply Resources/Textures/Decals/medical_decals.rsi/meta.json:4]: RSI Editor will autofill this if you paste the url of the github link you downloaded the .dmi at.

Comment (QuietlyWhisper) [reply Resources/Textures/Decals/medical_decals.rsi/meta.json:4]: https://discord.com/channels/1168210010233376858/1168210011823026271/1169807706123685968

**Comment by DrSmugleaf** [comment]
Thanks for the PR, I updated it to work with the new way that we can hide non-cm decals from the spawn menu in [this commit](https://github.com/CM-14/CM-14/commit/82449885145b5666226c7d9aea30795216ebc9bc) so that we don't need to delete any decals from upstream, this is what it looks like in-game:

image

**Pull request merged**